### DISCUSSION TITLE
Edit publish layout: add "move to [tab name]" dropdown

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2913] [Feature Requests]
It can be tough to drag and drop fields from the bottom of a long list to the tab you want to the field to be in. It's easy to miss your drop target, especially if you are having to drag and scroll at the same time.

**Describe the solution you'd like**
Alongside the "hide" and "collapse" options, a new select box that lets you select the tab you want to move the field to. Ideally, selecting a tab in the dropdown would *not* move your focus to that tab, allowing you to continue working on the list of fields already in front of you.

**Comment by ic360-com** [comment]
Nice idea - like this one. It'd save a lot of time dragging and dropping, for complex builds.
